src/toil_scripts/adam_uberscript/adam_uberscript.py

```diff
@@ -119,8 +118,13 @@ def launch_pipeline(params):
                                '-o', 'UserKnownHostsFile=/dev/null',
                                '-o', 'StrictHostKeyChecking=no',
                                'screen', '-dmS', params.cluster_name])
-        # Run command on screen session
-        
+        # Run command on screen session  
+
+        if params.reference_genome == 'GRCh38':
+            from toil_scripts.adam_uberscript.input_files import GRCh38_inputs as inputs
+        if params.reference_genome == 'hg19':
+            from toil_scripts.adam_uberscript.input_files import hg19_inputs as inputs       
```

Comment:
Yes, validate early and raise RuntimeException to alert user but **also** assert here. If someone changes the validation code and forgets to change this code here, they will have introduced a bug which would be caught by the assert. This is especially important considering Python's lack of type-safe enumerations. And -1 on informational message, the stack trace will be informative enough.